diff --git a/apple/scripts/build-xcframework.sh b/apple/scripts/build-xcframework.sh
--- a/apple/scripts/build-xcframework.sh
+++ b/apple/scripts/build-xcframework.sh
@@ -237,6 +237,7 @@ build_slice() {
     OBJROOT="${DERIVED_DATA_PATH}/Build/Intermediates.noindex" \
     BUILD_LIBRARY_FOR_DISTRIBUTION=YES \
     SKIP_INSTALL=NO \
+    CODE_SIGNING_ALLOWED=NO \
     DEBUG_INFORMATION_FORMAT=dwarf-with-dsym \
     COMPILER_INDEX_STORE_ENABLE=NO \
     SWIFT_COMPILATION_MODE=wholemodule \
